clusterloader2/testing/load/config.yaml

```diff
@@ -1,12 +1,33 @@
+#Constants
+{{$BIG_GROUP_SIZE := 250}}
+{{$MEDIUM_GROUP_SIZE := 30}}
+{{$SMALL_GROUP_SIZE := 5}}
+{{$PODS_PER_NODE := 30}}
+{{$NODES_PER_NAMESPACE := 100}}
+#Variables
+# bigRCs - 1/4 of all pods should be in big RCs.
+{{$bigRCs := MultiplyInt .Nodes 0.03}}
+# mediumRCs - 1/4 of all pods should be in medium RCs.
+{{$mediumRCs := MultiplyInt .Nodes 0.25}}
+# smallRCs - 1/2 of all pods should be in small RCs.
+{{$smallRCs := MultiplyInt .Nodes 3}}
+{{$namespaces := DivideInt (SubtractInt .Nodes 1) $NODES_PER_NAMESPACE | AddInt 1}}
+{{$bigRCsPerNamespace := DivideInt $bigRCs $namespaces}}
+{{$mediumRCsPerNamespace := DivideInt $mediumRCs $namespaces}}
+{{$smallRCsPerNamespace := DivideInt $smallRCs $namespaces}}
+{{$totalPods := MultiplyInt $PODS_PER_NODE .Nodes}}
+# saturationQPS - qps for RC operations should be an equivalent to 10 [pods/s].
```

Comment:
The problem is that this qps is calculated with assumption that we want to achieve 10pods/s. However, there is also service created for every RC. I applied x2, because there are always two objects that should be created.

Reply:
Maybe I should separate service creation from rc creation?

Reply:
I'm not sure I understand the relation b/w services and RC creations here. Throughput is calculated from the rate at which we want pods to be created (which is decided by the rate at which we create RCs).

Reply:
For every RC, one service is created. So I assumed that I should use bundle = [service, RC].
However, in the original load test, all services are created and then all the RC. The `AvgSaturationQPS` was originally only used for RC creation, so I needed to recalculated it for svc + RC bundle. That's why I used x2 multiplier.

Maybe it was my mistake to use bundle and I should use original test logic.

Reply:
Bundling service and RC together is the right decision.

But I also don't understand why the throughput was changed - the throughput as Shyam mentioned above is for pods and whether we create a service or not is orthogonal.

Reply:
Cause after this change (making bundle), total number of pods remained the same and number of operations (for object creation) that need tick from tuning set increased two times.
To keep the same expected time for pod creation, I had to increase qps two times - half of the ticks are consumed by the services.

Reply:
Why isn't one tick used per object bundle?

Reply:
yes - that sounds liek a bug.

Reply:
Tuning set is all about operation per seconds. Inside the object bundle you can have any number of objects (operation). The real operations per seconds would be tuning set qps + all operations from any started bundles that haven't finished yet.

I think that tick per object operation approach is easier to understand and better implements the idea behind tuning sets.

Reply:
> Tuning set is all about operation per seconds.

I'm not sure I agree with that because TuningSet is given per object bundle as opposed to per object (which kind of makes sense given that the bundle forms a logical unit of user workload). @wojtek-t - wdyt?

Reply:
+1 to Shyam - your "unit" in ClusterLoader is object bundle and that is what we are executing.

Reply:
I stand corrected then. I always thought that object bundle was introduced only to ensure order of object operations :/

Reply:
I created #223 to fix the ticker.

Reply:
I've changed this number from `20` to `10`.

Reply:
#223 merged.
@shyamjvs @wojtek-t Friendly ping to keep PR alive.